pkgs/dartpad_ui/lib/main.dart

```diff
@@ -59,10 +59,6 @@ class _DartPadAppState extends State<DartPadApp> {
         path: '/',
         builder: _homePageBuilder,
       ),
-      GoRoute(
```

Comment:
@johnpryan should likely review this - he just landed a PR that introduces a `:gistId` route: https://github.com/dart-lang/dart-pad/pull/3096 (I believe this restores functionality from before the rewrite).